packages/web-integration-oc10/lib/Controller/ConfigController.php

```diff
@@ -124,4 +125,20 @@ private function addOC10AppsToConfig(array $config): array {
         $config['applications'] = $apps;
         return $config;
     }
+
+    /**
+     * Add an account edit link to the config.
+     *
+     * @param array $config
+     * @return array
+     */
+    private function addAccountEditLinkToConfig(array $config): array {
+        $options = $config['options'] ?? [];
+        $serverUrl = $this->request->getServerProtocol() . '://' . $this->request->getServerHost();
+        $options['accountEditLink'] = [
```

Comment:
This should only be done if the `config.json` loaded from disk didn't specify it on its own. I.e. if `acountEditLink` already exists you can return early.

Reply:
see https://github.com/owncloud/web/pull/8294/commits/96363f669b22a3a55a9609cf0fc321458656cd92